app/uk/gov/hmrc/customs/notification/services/UnblockPollerService.scala

```diff
@@ -42,65 +42,83 @@ class UnblockPollerService @Inject()(config: CustomsNotificationConfig,
     val pollerInterval: FiniteDuration = config.unblockPollerConfig.pollerInterval
 
     actorSystem.scheduler.scheduleWithFixedDelay(0.seconds, pollerInterval) { () => {
-      notificationWorkItemRepo.distinctPermanentlyFailedByCsId().map { permanentlyFailedCsids: Set[ClientSubscriptionId] =>
-        logger.info(s"Unblock - discovered ${permanentlyFailedCsids.size} blocked csids (i.e. with status of ${PermanentlyFailed.name})")
-        logger.debug(s"Unblock - discovered $permanentlyFailedCsids blocked csids (i.e. with status of ${PermanentlyFailed.name})")
-        permanentlyFailedCsids.foreach { csid =>
-          notificationWorkItemRepo.pullOutstandingWithPermanentlyFailedByCsId(csid).map {
-            case Some(workItem) =>
-              pushOrPull(workItem).foreach(ok =>
-                if (ok) {
-                  // if we are able to push/pull we flip statues from PF -> F for this CsId by side effect - we do not wait for this to complete
-                  //changing status to failed makes the item eligible for retry
-                  notificationWorkItemRepo.fromPermanentlyFailedToFailedByCsId(csid).foreach { count =>
-                    logger.info(s"Unblock - number of notifications set from PermanentlyFailed to Failed = $count for CsId ${csid.toString}")
-                  }
-                }
-              )
-            case None =>
-              logger.info(s"Unblock found no PermanentlyFailed notifications for CsId ${csid.toString}")
-          }
+      notificationWorkItemRepo.distinctPermanentlyFailedByCsId()
+        .foreach { permanentlyFailedCsids: Set[ClientSubscriptionId] =>
+          logger.info(s"Unblock - discovered ${permanentlyFailedCsids.size} blocked csids (i.e. with status of ${PermanentlyFailed.name}: $permanentlyFailedCsids)")
```

Comment:
Could the logs have [] around the variables logged out
e.g.
Unblock - discovered [3] blocked...

Reply:
Done 👍🏼